__main__.py

```diff
@@ -4,43 +4,99 @@
 Usage: run the exerciser with a variety of options
 """
 
-from pathlib import Path
 import argparse
-from datetime import datetime
 import sys
 import os
 
 sys.path.append(os.path.join(os.path.dirname(__file__), "src"))
 from src import general
 
+
 def parse_cla() -> argparse.Namespace:
     """
     Usage: command line argument parser
     @return: parsed arguments in argparse.Namespace object
     """
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("test_name",
-                        type=str,
-                        metavar="test_name",
-                        choices=["sleep_test", "checksum_test"],
-                        help="name of test to be run by exerciser")
+    parser.add_argument(
+        "tests",
+        nargs="*",
+        default=[],
+        help="specify the test/s to run in a space seperated list. if no tests "
+        + "are listed, the exerciser will run all available tests.",
+    )
+
+    parser.add_argument(
+        "-f",
+        "--flush-all",
+        action="store_true",
+        dest="flush_all",
+        help="deletes all subdirectories in the working dir. if this option is used with "
+        + "-w, it will flush the contents of that dir rather than the default working dir.",
```

Comment:
Consider rephrasing to `Clears out all previous exerciser execution subdirectories from the specified working directory`